packages/rlc-common/src/interfaces.ts

```diff
@@ -141,6 +141,9 @@ export interface Schema {
   default?: any;
   readOnly?: boolean;
   usage?: SchemaContext[];
+  alias?: string;
+  outputAlias?: string;
```

Comment:
I feel like our alias logic is sperated into different places, sometimes in schema, sometimes in responses.

I smelled that we need to refactor the alias logic if the modelUtil could have it, it's better to move necessary logic into there.

Reply:
when do we use alias in response ?

Reply:
Maybe I remember wrongly but we have alias model in merge+path, right?

Reply:
sync offline, would like keep merge and patch as it is.